gradle/versions.gradle

```diff
@@ -43,7 +43,7 @@ ext {
     powerMockVersion = "2.0.9"
     runnerVersion = "1.2.0"
     rulesVersion = "1.2.0"
-    robolectricVersion = "4.9.2"
+    robolectricVersion = "4.11.1"
```

Comment:
Do we need this?

Reply:
made robolectric sdk upgrade to run the UTs in monthly pipeline smoothly